### ISSUE TITLE
4.1.1 Parsing is missing the WCAG 2.2 content in the built document

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#345]
Not sure when this happened, but I just noticed that 4.1.1 Parsing's quoted WCAG 2.2 content is blank rather than having the single note: 
>NOTE: This criterion was originally adopted to address problems that assistive technology had directly parsing HTML. Assistive technology no longer has any need to directly parse HTML. Consequently, these problems either no longer exist or are addressed by other criteria. This criterion no longer has utility and is removed.

So it seems there is some scripting issue or the scraped content from the WCAG repository is once again old and needs to be regenerated.

**Comment by maryjom** [comment]
@daniel-montalvo Fixed with PR #347. Thanks Daniel!
